test/support/data_case.ex

```diff
@@ -32,6 +32,10 @@ defmodule Skate.DataCase do
       Ecto.Adapters.SQL.Sandbox.mode(Skate.Repo, {:shared, self()})
     end
 
+    on_exit(fn ->
+      Ecto.Adapters.SQL.Sandbox.checkin(Skate.Repo)
```

Comment:
what's this about now?

Reply:
Each time you check out a sandbox it runs everything within a transaction. Previously, we weren't checking them in in between tests, so changes from one test would accumulate and potentially interfere with another test. The options were to have each test I was writing manually delete the contents of tables that it cared about before doing anything, or to just make it so that each test gets a copy of the DB unaffected by the changes from other tests.